util/chplenv/third_party_utils.py

```diff
@@ -15,12 +15,11 @@
 #
 @memoize
 def default_uniq_cfg_path():
-    arch_val = chpl_arch.get('target', map_to_compiler=True,
-                             get_lcd=using_chapel_module()).arch
-    return '{0}-{1}-{2}-{3}'.format(chpl_platform.get('target'),
-                                    chpl_compiler.get('target'),
-                                    arch_val,
-                                    chpl_lib_pic.get())
+    pic = chpl_lib_pic.get()
+    result = chpl_bin_subdir.get('target')
```

Comment:
Hmm, Calling `bin_subdir` to get the the library_subdir is a little odd. Maybe we can change it to chpl_unique_subdir or something?

Reply:
I just stopped doing that. Even if they're similar I think at the moment it's better to not make them call each other.